wremnants/qcdScaleByHelicity_helper.py

```diff
@@ -18,7 +18,17 @@
 
 def makeQCDScaleByHelicityHelper(is_w_like = False, filename=None):
     if filename is None:
-        filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs.pkl.lz4"
+        #
+        # FIXME: for tests with binning, to be removed eventually
+        #
+        # if is_w_like:
+        #     filename = f"{data_dir}/angularCoefficients/z_coeffs.pkl.lz4"
+        #     filename = f"{data_dir}/angularCoefficients/z_coeffs_testVpt.pkl.lz4"
+        # else:
+        #     #filename = f"{data_dir}/angularCoefficients/w_coeffs.pkl.lz4"
+        #     filename = f"{data_dir}/angularCoefficients/w_coeffs_testVpt.pkl.lz4"
+        #filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs.pkl.lz4"
+        filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs_testVpt.pkl.lz4" # Vpt binning based on 10% quantiles (but from old aMC@NLO)
```

Comment:
Why does this need to be modified? There's no reason the binning in the analysis hist has to match the one for the coeffs

Reply:
Perhaps I am wrong, but isn't it the file that is used to create the binning for the QCD scale histogram? It's exactly there that I wanted these 10 bins in Wpt. The problem then is that the binning is not a subset of all natural numbers, so I can't simply rein afterwards to get the desired quantiles (although one could use approximate quantiles)

Reply:
It is currently but it doesn't have to be

Reply:
fixed by the new commit [1f75b3e](https://github.com/WMass/WRemnants/pull/71/commits/1f75b3ef9eaaf3c6b174362ab1df5fb618ebb267)